src/interface/src/app/plan/create-scenarios/create-scenarios.component.ts

```diff
@@ -64,7 +64,11 @@ export class CreateScenariosComponent implements OnInit, OnDestroy {
   panelExpanded: boolean = true;
   stepStates: StepState[];
   treatmentGoals: Observable<TreatmentGoalConfig[] | null>;
-
+  defaultQ: TreatmentQuestionConfig = {
```

Comment:
I know it sounds silly, but write out Question.

Reply:
Whoops thanks for catching that!